src/Dev/TestMailer.php

```diff
@@ -2,88 +2,30 @@
 
 namespace SilverStripe\Dev;
 
-use SilverStripe\Control\Email\Mailer;
+use SilverStripe\Control\Email\Email;
+use SilverStripe\Control\Email\SwiftMailer;
 
-class TestMailer extends Mailer
+class TestMailer extends SwiftMailer
 {
-    protected $emailsSent = array();
-
     /**
-     * Send a plain-text email.
-     * TestMailer will merely record that the email was asked to be sent, without sending anything.
-     *
-     * @param string $to
-     * @param string $from
-     * @param string $subject
-     * @param string $plainContent
-     * @param bool $attachedFiles
-     * @param bool $customHeaders
-     * @return bool|mixed
+     * @var array
      */
-    public function sendPlain($to, $from, $subject, $plainContent, $attachedFiles = false, $customHeaders = false)
-    {
-        $this->saveEmail([
-            'Type' => 'plain',
-            'To' => $to,
-            'From' => $from,
-            'Subject' => $subject,
-
-            'Content' => $plainContent,
-            'PlainContent' => $plainContent,
-
-            'AttachedFiles' => $attachedFiles,
-            'CustomHeaders' => $customHeaders,
-        ]);
-
-        return true;
-    }
+    protected $emailsSent = array();
 
     /**
-     * Send a multi-part HTML email
-     * TestMailer will merely record that the email was asked to be sent, without sending anything.
-     *
-     * @param string $to
-     * @param string $from
-     * @param string $subject
-     * @param string $htmlContent
-     * @param bool $attachedFiles
-     * @param bool $customHeaders
-     * @param bool $plainContent
-     * @param bool $inlineImages
-     * @return bool|mixed
+     * @param \SilverStripe\Control\Email\Email $message
+     * @return int
      */
-    public function sendHTML(
-        $to,
-        $from,
-        $subject,
-        $htmlContent,
-        $attachedFiles = false,
-        $customHeaders = false,
-        $plainContent = false,
-        $inlineImages = false
-    ) {
-
-        $this->saveEmail([
-            'Type' => 'html',
-            'To' => $to,
-            'From' => $from,
-            'Subject' => $subject,
-
-            'Content' => $htmlContent,
-            'PlainContent' => $plainContent,
-            'HtmlContent' => $htmlContent,
-
-            'AttachedFiles' => $attachedFiles,
-            'CustomHeaders' => $customHeaders,
-            'InlineImages' => $inlineImages,
-        ]);
+    public function send($message)
+    {
+        $this->saveEmail($message);
```

Comment:
But if they rely on some of the specific data (like `Type`) then behat tests will need changing because that's not really a thing anymore.... or we'll need to have a "best guess" at type.